src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AssessmentTransformer.kt

```diff
@@ -111,20 +111,23 @@ class AssessmentTransformer(
   }
 
   fun getSortedReferralHistoryNotes(
-    jpa: TemporaryAccommodationAssessmentEntity,
+    assessment: TemporaryAccommodationAssessmentEntity,
     cas3Events: List<DomainEventEntity>,
+    filterUserNotes: Boolean = false,
```

Comment:
in the kotlin `filter` closure we use 'true' to mean 'include in the response'. I think this variable is using 'true' to mean 'don't include in the response.

would a more consistent name be 'includeUserNotes' (and flip the boolean?). 

but it's possible i'm getting my booleans confused :)